agent/otel.go

```diff
@@ -42,6 +43,7 @@ const (
 	packageName    = "github.com/evergreen-ci/evergreen/agent"
 	traceSuffix    = "build/OTelTraces"
 	maxLineSize    = 1024 * 1024
+	dataMountpoint = "/data"
```

Comment:
Since this comes from a distro setting, rather than hardcoding our assumed distro settings into the agent implementation, can we have this be a distro setting and pull this from the distro info? Having the filesystem info live in one place would be nice.

Reply:
done